ztp/siteconfig-generator-kustomize-plugin/testSiteConfig/site1-sno-du-tpm2-nbde.yaml

```diff
@@ -0,0 +1,122 @@
+apiVersion: ran.openshift.io/v2
+kind: SiteConfig
+metadata:
+  name: "site-sno-du-1-tpm2-nbde"
+  namespace: "site-sno-du-1-tpm2-nbde"
+spec:
+  baseDomain: "example.com"
+  pullSecretRef:
+    name: "pullSecretName"
+  clusterImageSetNameRef: "openshift-v4.13.0"
+  sshPublicKey: "ssh-rsa "
+  sshPrivateKeySecretRef:
+    name: "sshPrvKey"
+  clusters:
+  - clusterName: "site-sno-du-1-tpm2-nbde"
+    biosConfigRef:
+      filePath: "testSiteConfig/testHW.profile"
+    networkType: OVNKubernetes
+    installConfigOverrides:  "{\"capabilities\":{\"baselineCapabilitySet\": \"None\", \"additionalEnabledCapabilities\": [ \"marketplace\", \"NodeTuning\" ] }}"
+    extraManifestPath: testSiteConfig/testUserExtraManifest
+    clusterLabels:
+      group-du-sno: ""
+      common: true
+      sites : "site-sno-du-1"
+    clusterNetwork:
+      - cidr: 10.128.0.0/14
+        hostPrefix: 23
+    machineNetwork:
+      - cidr: 10.16.231.0/24
+    serviceNetwork:
+      - 172.30.0.0/16
+    manifestsConfig:
+      ntpServer: "pool.ntp.org"
+    additionalNTPSources:
+      - NTP.server1
+      - 10.16.231.22
+    ignitionConfigOverride: "igen"
```

Comment:
I think this file should contain ONLY the disk encryption stuff
kustomize will bring the rest from the other files...

Reply:
fixed